### ISSUE TITLE
Chat - For quote, live markdown is not shown

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#42755]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.77-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N
**If this was caught during regression testing, add the test name, ID and link from TestRail:** https://expensify.testrail.io/index.php?/tests/view/4582241
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Tap on a report
3. Enter >hai
4. Send the message
5. Long tap and open edit comment

## Expected Result:
For quote, live markdown must be shown

## Actual Result:
For quote, live markdown is not shown but in edit comment markdown is shown.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [ ] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/cfbdc945-1075-411b-bd38-5185cd37c8f0

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by izarutskaya** [comment]
@abekkala I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors.

**Comment by izarutskaya** [comment]
We think this issue might be related to the #vip-vsb

**Comment by izarutskaya** [comment]
Production

https://github.com/Expensify/App/assets/115492554/f21df9cf-4f3a-42e1-87f1-775ae039a976

**Comment by Julesssss** [comment]
This shouldn't block the app, it's a minor UI issue, likely introduced as part of the recent [live-markdown library bump](https://github.com/Expensify/App/pull/42454)

**Comment by Julesssss** [comment]
I shared this in the live markdown Slack channel [here](https://expensify.slack.com/archives/C06BDSWLDPB/p1716986227425769)

**Comment by Skalakid** [comment]
Hi, I'm Michał from SWM and I will investigate this problem

**Comment by Skalakid** [comment]
After quick investigation I found out that live markdown parser is using different commit of `expensify-common` than E/App. Live Markdown parser is using newer version of ExpeniMark that contain changes that block `blockquote` creation without space after `>`. I will create PR that bumps the version in Expensify App

**Comment by Skalakid** [comment]
I pulled the latest main and E/App has newer version of `expensify-common` than live markdown. I can't reproduce this issue on the latest main now

**Comment by Julesssss** [comment]
Ah thanks for taking a look. Let me confirm

**Comment by Julesssss** [comment]
@izarutskaya, is this bug report for the case where no space is left between the chevron and text? For example: 

```
>text
```

**Comment by Skalakid** [comment]
@Julesssss the problem is with the representation of the chevron and text inside the input and message. However, I'm not able to reproduce this issue on the latest main. Let me know if it works on your end

**Comment by Julesssss** [comment]
I agree that's the reported problem, but I think the distinction is between `>test` and `> test`. 

`>test` works for me in production, but not staging and dev
`> test` works in all environments

**Comment by Skalakid** [comment]
Yes, and `>test` should not work, it's a new change. I'm not sure if this issue is reproducible, let me know if it's working on your end. The only change that we can do is to update expensify-common in E/App and `react-native-live-markdown`, but we will do it together with code block PR

**Comment by fabOnReact** [comment]
I would be happy to work on this task. Thanks

**Comment by Julesssss** [comment]
Closing, this was an expected change
